server/routes/analyticsRouter.test.ts

```diff
@@ -50,30 +50,30 @@ describe('Home page shows card linking to incentives analytics', () => {
   it('if feature is turned on', () => {
     return request(app)
       .get('/')
-      .expect(res => expect(res.text).toContain('Incentives data'))
+      .expect(res => expect(res.text).toContain('/analytics'))
```

Comment:
look for URL rather than tile text so that it can vary without tests needing to be changed each time